README.md

```diff
@@ -1,27 +1,49 @@
-# epm
-
-Ethereum Package Managment
+# EPM
 
 [![Join the chat at https://gitter.im/ethpm/Lobby](https://badges.gitter.im/ethpm/Lobby.svg)](https://gitter.im/ethpm/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-# Goals
+Ethereum Packaging Specification
```

Comment:
> EVM Smart Contract Packaging Specification
👍 
Since indeed the use of packages could be used by different EVM chains like Rootstock